include/engine/routing_algorithms/routing_base_mld.hpp

```diff
@@ -54,8 +55,10 @@ inline bool checkParentCellRestriction(CellID, const PhantomNodes &) { return tr
 // Restricted search (Args is LevelID, CellID):
 //   * use the fixed level for queries
 //   * check if the node cell is the same as the specified parent onr
-inline LevelID
-getNodeQueryLevel(const partition::MultiLevelPartitionView &, NodeID, LevelID level, CellID)
+inline LevelID getNodeQueryLevel(const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &,
```

Comment:
Leave the previous formatting and replace ContiguousInternalMemoryDataFacade with AlgorithmDataFacade